_posts/2018-01-22-LGBTSteminar2018.md

```diff
@@ -0,0 +1,23 @@
+---
+layout:     post
+title:      LGBT STEMinar
+date:       2018-01-22 09:00
```

Comment:
This should be in the future, of course, unless we do want to backdate the post?

Reply:
will update to the future when ready to publish

Reply:
Think we're good to go 👍